app/src/main/java/com/sdp13epfl2021/projmag/database/di/UserdataDatabaseModule.kt

```diff
@@ -16,10 +16,10 @@ import javax.inject.Singleton
 object UserdataDatabaseModule {
     @Provides
     @Singleton
-    fun bindUserdataDatabase(
+    fun providesUserdataDatabase(
         userDB: FirebaseUserdataDatabase,
-        @Named("userID") uid: String,
-        @Named("userRootDir") dir: File
+        @Named("currentUserId") uid: String,
+        @Named("fileCacheUserDB") dir: File
```

Comment:
Maybe avoid string literals (especially for those you use in more than one place)